free-1.md

```diff
@@ -10,34 +10,46 @@ _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help 
 them?_
 
+Part of being a great drummer is having a wide variety of interesting drum fills in your arsenal.  However, since drummers' ears/hands are so used to hearing/playing the same cliche drum fills, it can be difficult to come up with original, rhythmically-interesting fills.  With some sort of external creative stimulation, this could become much easier.
 
```

Comment:
Mentioning what a drum fill is might help, I had no idea, but was able to figure out eventually by context.